### DISCUSSION TITLE
VS Code version

### DISCUSSION THREAD

**Discussion by jdiclemente** [coder/code-server#2005] [Q&A] [answered]
I am running code-server: v3.4.1 with VS Code: v1.45.1. I am not able to run mssql extension because of the version. How can I update VS Code version.

**Comment by bpmct** [comment, marked as answer]
It looks like the VS Code version has been updated (#2000), but there has not been a release yet.

For now, you can [build code-server](https://github.com/cdr/code-server/blob/v3.4.1/doc/CONTRIBUTING.md#build) or use an [older version of mssql](https://github.com/microsoft/vscode-mssql/releases) which would likely get this working.
